stdlib/src/utils/static_tuple.mojo

```diff
@@ -400,3 +400,11 @@ struct InlineArray[ElementType: CollectionElement, size: Int](Sized):
         return Reference(self)[]._get_reference_unsafe[mutability, self_life](
             normalized_idx
         )
+
+    fn get_storage_unsafe_pointer(self) -> UnsafePointer[ElementType]:
```

Comment:
We explicitly chose not to reveal this in the safe API in the original PR. If you _really_ think we should be adding it, we should at least prefix it with an `_`, but the general stance is that getting an unsafe pointer is not recommended.

Reply:
There are wayyyy too many places in the stdlib that uses a string pointer, so I don't mind removing it but this PR is not the place to do it.

Concerning the naming, I'm just following the instructions given in the contributing guide. There is nothing stating that unsafe functions should be private, but we should be explicit about the unsafe part (like in swift):
https://github.com/modularml/mojo/blob/main/stdlib/docs/vision.md#objectives-and-goals

Reply:
Drive-by comment: we have an internal PR which is harmonizing _all_ the "get an unsafe pointer" APIs to `unsafe_ptr()`.

Reply:
It's pretty rare in python to abbreviate words, but I guess that folks coming from low level languages too used to write "ptr" everywhere XD I don't want to bikeshed too much, but just saying that there is no reason to use "ptr" instead of "pointer"

Reply:
Right, I don't feel strongly on the eventual unified/final name.  Mostly care about unifying the various different namings we have now across the types.